integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/apim/utils/AdminUtils.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.choreo.connect.tests.apim.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.am.integration.clients.admin.api.AdvancedPolicyCollectionApi;
+import org.wso2.am.integration.clients.admin.api.ApplicationPolicyCollectionApi;
+import org.wso2.am.integration.clients.admin.api.SubscriptionPolicyCollectionApi;
+import org.wso2.am.integration.clients.admin.api.dto.*;
```

Comment:
Let's import only the required set.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/2736/commits/7048d5c96997f9537de523dba580cdd561454894